exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark > 50) {
+    return "C";
+  } else {
+    return "F";
```

Comment:
This is correct, but you are repeating some checks that are redundant. For example, on line 12 you already check `if (mark >= 80)`. This means that on line 14 you don't need to check if `mark < 80`. It _must_  be, because if it weren't less than 80, then the code on line 13 would have executed, and you'd never have reached line 14.

So, in short, this function would do exactly the same thing:

```
function calculateGrade(mark) {
  if (mark >= 80) {
    return "A";
  } else if (mark > 60) {
    return "B";
  } else if (mark > 50) {
    return "C";
  } else {
    return "F";
  }
}
```